src/App.tsx

```diff
@@ -325,17 +413,36 @@ function App() {
   }
 
   return (
-    <div className="grid grid-cols-4 grid-rows-1 gap-4">
+    <div className='grid grid-cols-4 grid-rows-1 gap-4'>
```

Comment:
I guess this is prettier's doing? We should probably add that to the husky runner. Could you make an issue?

Reply:
No that was actually me but I will make an issue for the prettier.